### PR TITLE
Feat add by-age-range

### PR DESCRIPTION
**Pull request by jalezi**
see also #3 and #4

At the moment only for range "90+".

### CODE REVIEW

File: cepimose/by_age_range/types.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+@dataclass
+class VaccinationDose:
+    date: datetime.datetime
+    dose: int
+
+
+@dataclass
+class VaccinationByAgeRange:
+    dose1: list[VaccinationDose]
```
Comment (overlordtm) [inline cepimose/by_age_range/types.py:13]: put single quotes around type `dose1: 'list[VaccinationDose]'`

File: test/test.py [hunk 2]
```diff
@@ -17,11 +17,12 @@ def test_vaccinations_by_day(self):
 
         def assertRow(row, expected_date, expected_first, expected_second):
             self.assertEqual(row.date, expected_date)
-            self.assertEqual(row.first_dose, expected_first)
-            self.assertEqual(row.second_dose, expected_second)
+            self.assertAlmostEqual(row.first_dose, expected_first, delta=10)
```
Comment (overlordtm) [inline test/test.py:20]: delta=30 or something bigger. Numbers change everyday :D

Comment (overlordtm) [reply test/test.py:20]: Everywhere :D

File: test/test.py [hunk 3]
```diff
@@ -113,3 +113,21 @@ def assertRow(row, expected_date, expected):
...
+        def assertRow(row, expected_date, expected):
+            self.assertEqual(row.date, expected_date)
+            self.assertEqual(row.pfizer, expected[0])
+            self.assertEqual(row.moderna, expected[1])
+            self.assertEqual(row.az, expected[2])
+
+        assertRow(data[3], datetime.datetime(2021, 1, 11), [59670, None, None])
+        assertRow(data[7], datetime.datetime(2021, 1, 31), [None, 3600, None])
+        assertRow(data[10], datetime.datetime(2021, 2, 6), [None, None, 9600])
+        assertRow(data[16], datetime.datetime(2021, 2, 25), [None, 16800, 52800])
```
Comment (jalezi) [inline test/test.py:130]: What about this test?

**Pull request merged**